ush/forecast_postdet.sh

```diff
@@ -2,12 +2,14 @@
 
 # Disable variable not used warnings
 # shellcheck disable=SC2034
+# shellcheck disable=SC2178
```

Comment:
Where is an array being overwritten by a string?  This seems like a check we should keep and disable on a case-by-case basis as assumptions may be made about whether a variable is an array or not.